src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -74,7 +74,6 @@ namespace cryptonote
       END_KV_SERIALIZE_MAP()
     };
   };
-
```

Comment:
don't alter either indents or line lines. the patch should affect as less as possible